app/public/css/style.css

```diff
@@ -0,0 +1,231 @@
+  padding: 50px;
+  background-color: #fff !important;
+}
+
+.title {
+  padding: 30px;
+  color: #565a5c !important;
+}
+
+.logo {
+  font-family: 'Nunito', sans-serif;
+}
+
+.navbar {
+  background-color: #fb4743 !important;
+  border: none !important;
+  opacity: 0.8;
+}
+
+.navbar-brand {
+  font-family: 'Nunito', sans-serif;
+  color: #fff !important;
+}
+
+.btn {
+  margin: 5px ;
+  border: 1px solid #e6ebe0 !important;
+  border-radius: 2px !important;
+  background-color: #fff;
+  color: #fb4743;
+}
+
+.btn:hover {
+  border: none;
+  color: #fff !important;
+  background-color: #fb4743;
+  opacity: 0.85;
+}
+
+.nav-btn {
+  color: #fff !important;
+  background-color: #fb4743 !important;
+  border: none !important;
+}
+
+.nav-btn:hover {
+  color: #ddd !important;
+}
+
+.sign-up-btn {
+  border: 1px solid #e6ebe0 !important;
+}
+
+.sign-up-btn:hover {
+  background-color: #fc7e7b !important;
+  color: #fff !important;
+}
+
+.center {
+  text-align: center;
+  width: 100%;
+  margin: 0 auto;
+}
+
+h3 {
+  padding: 10px;
+}
+
+label {
+  color: #555;
+  padding: 5px;
+}
+
+textarea {
+  resize: vertical;
+}
+
+.list {
+  list-style: none;
+  margin: 10px;
+  padding: 20px;
+  border: 1px solid #ccc;
+  background-color: #fff;
+}
+
+.list:last-child {
+  margin-bottom: 50px;
+}
+
+.name, .price {
+  font-size: 16px;
+  font-weight: bold;
+}
+
+a {
+  text-decoration: none !important;
+}
+
+p {
+  color: #444;
+}
+
+#to-top {
+  position: relative;
+  bottom: 30px;
+  left: 50%;
+  font-size: 18px;
+}
+
+#to-top a {
+  color: #bbb;
+}
+
+#to-top a:hover {
+  color: #999;
+}
+
+#notice {
+  text-align: center;
+}
+
+#errors {
+  text-align: center;
+  list-style: none;
+  color: #fb4743;
+}
+
+.ui-datepicker {
+  background-color: #fff;
+  padding: 10px;
+  font-family: 'Lato', sans-serif !important;
+  font-size: 18px !important;
+  height: auto;
+  width: 260px;
+  margin: 5px auto 0;
+  cursor: pointer;
+  position: relative;
+  top: 30px;
+}
+
+.ui-datepicker table {
+  width: 100%;
+}
+
+.ui-datepicker-header {
+  color: #fb4743;
+  opacity: 0.8;
+}
+
+.ui-datepicker-month {
+  ;
+}
+
+.ui-datepicker-title {
+  text-align: center;
+}
+
+.ui-datepicker-prev {
+  color: #fb4743;
+  opacity: 0.8;
+  float: left;
+  background-position: center -30px;
+}
+
+.ui-datepicker-next {
+  color: #fb4743;
+  opacity: 0.8;
+  float: right;
+  background-position: center 0px;
+}
+
+.ui-datepicker th {
+  background-color: #f7f7f7;
+  text-align: center;
+}
+
+.ui-datepicker tbody td {
+   text-align: center;
+}
+
+.ui-datepicker td span, .ui-datepicker td a {
+    display: inline-block;
+    font-weight: bold;
+    text-align: center;
+    width: 30px;
+    height: 30px;
+    line-height: 30px;
+    color: #666666;
+    text-shadow: 1px 1px 0px #fff;
+    filter: dropshadow(color=#fff, offx=1, offy=1);
+}
+
+.ui-datepicker-unselectable .ui-state-default {
+    color: #b4b3b3;
+}
+
+.ui-datepicker-calendar .ui-state-hover {
+   color: #bbb;
+}
```

Comment:
Extremely clear and well laid-out CSS. I would suggest separating components into different files, but this is really nice :+1: